src/Proto.Cluster/Cluster.cs

```diff
@@ -22,6 +22,8 @@ public class Cluster
 
         public Cluster(ActorSystem system, ClusterConfig config)
         {
+            system.RegisterClusterPlugin(this);
```

Comment:
create a cluster plugin that knows about this cluster, register this plugin for the system